### ISSUE TITLE
Installation instructions are not clear

### ISSUE THREAD

**Issue by petersmythe** [geostyler/geostyler-geoserver-plugin#93]
On https://github.com/geostyler/geostyler-geoserver-plugin#installation- , the instuctions say:

> Download the matching jar from [here](https://github.com/geostyler/geostyler-geoserver-plugin/packages).

Following that link shows a single package.  I guess it is obvious to click through.

Instead of finding the "matching jar", the next page shows maven instructions.  On the right are "Assets" - it is not obvious that the required JAR file is here, amongst the 30 files.

In a narrow column, with words wrapping, it is not easy to find the required file.  It is better like this:

```
gs-geostyler-1.1.1-20230313.135501-2-test-sources.jar.md5
gs-geostyler-1.1.1-20230313.135501-2-test-sources.jar.sha1
gs-geostyler-1.1.1-20230313.135501-2-test-sources.jar
gs-geostyler-1.1.1-20230313.135501-2-sources.jar.md5
gs-geostyler-1.1.1-20230313.135501-2-sources.jar.sha1
gs-geostyler-1.1.1-20230313.135501-2-sources.jar
gs-geostyler-1.1.1-20230313.135501-2-tests.jar.md5
gs-geostyler-1.1.1-20230313.135501-2-tests.jar.sha1
gs-geostyler-1.1.1-20230313.135501-2-tests.jar
gs-geostyler-1.1.1-20230313.135501-2.pom.md5
gs-geostyler-1.1.1-20230313.135501-2.pom.sha1
gs-geostyler-1.1.1-20230313.135501-2.pom
gs-geostyler-1.1.1-20230313.135501-2.jar.md5
gs-geostyler-1.1.1-20230313.135501-2.jar.sha1
gs-geostyler-1.1.1-20230313.135501-2.jar
gs-geostyler-1.1.1-20230313.132443-1-test-sources.jar.md5
gs-geostyler-1.1.1-20230313.132443-1-test-sources.jar.sha1
gs-geostyler-1.1.1-20230313.132443-1-test-sources.jar
gs-geostyler-1.1.1-20230313.132443-1-sources.jar.md5
gs-geostyler-1.1.1-20230313.132443-1-sources.jar.sha1
gs-geostyler-1.1.1-20230313.132443-1-sources.jar
gs-geostyler-1.1.1-20230313.132443-1-tests.jar.md5
gs-geostyler-1.1.1-20230313.132443-1-tests.jar.sha1
gs-geostyler-1.1.1-20230313.132443-1-tests.jar
gs-geostyler-1.1.1-20230313.132443-1.pom.md5
gs-geostyler-1.1.1-20230313.132443-1.pom.sha1
gs-geostyler-1.1.1-20230313.132443-1.pom
gs-geostyler-1.1.1-20230313.132443-1.jar.md5
gs-geostyler-1.1.1-20230313.132443-1.jar.sha1
gs-geostyler-1.1.1-20230313.132443-1.jar
```

There appear to be 2 releases above `-1` and `-2`?  

The last file in the list appears to be the required JAR file.

**Comment by jansule** [comment]
Thanks for the info. @dnlkoch could you take a look at this?

**Comment by dnlkoch** [comment]
Thanks @petersmythe, thanks for your feedback!

I agree that the presentation of the JARs in the GitHub package manager can become quite confusing (and has indeed some additional drawbacks). Anyway, I'll try to enhance the guideline so it might become easier to find the "matching jar".

TLDR:

- Go to the [latest version of the plugin](https://github.com/geostyler/geostyler-geoserver-plugin/packages/1469140?version=1.1.0) (actually 1.1.0).
- Find the `gs-geostyler-1.1.0.jar` link under `Assets` (on the right) and trigger the download by clicking on it.

**Comment by KaiVolland** [comment]
@petersmythe do you think this is solved with https://github.com/geostyler/geostyler-geoserver-plugin/pull/105? If yes please close the issue.

**Comment by petersmythe** [comment]
That's now brilliant!  Thank you for the massive improvement.
